### PR TITLE
add steps after open windows store

### PR DESCRIPTION
**Pull request by superskrem**
these steps was missing

### CODE REVIEW

**Comment by SuzeShardlow** [comment]
@slorello89 would you mind taking a look at this please?  It looks like something you know about! :)

File: docs/create/windows/index-windows.mdx [hunk 1]
```diff
@@ -30,6 +30,8 @@ Reboot Windows after making the change — note that you only need to do this on
 ```powershell
  start ms-windows-store:
 ```
+Then download the latest Ubuntu at this time its 20..4 LST.
```
Comment (slorello89) [inline docs/create/windows/index-windows.mdx]:
```suggestion

Then search for Ubuntu, or your preferred distribution of Linux, and download the latest version. 
```

**Review by slorello89 — CHANGES_REQUESTED** [review]

File: docs/create/windows/index-windows.mdx [hunk 2]
```diff
@@ -30,6 +30,8 @@ Reboot Windows after making the change — note that you only need to do this on
 ```powershell
  start ms-windows-store:
 ```
+Then download the latest Ubuntu at this time its 20..4 LST. 
+After it download run it and make sure it finish instlation.
```
Comment (SuzeShardlow) [inline docs/create/windows/index-windows.mdx]:
```suggestion
```

**Review by SuzeShardlow — APPROVED** [review]
Thanks @slorello89, I deleted the line after yours as it seemed superfluous.

**Review by SuzeShardlow — APPROVED** [review]

**Review by slorello89 — APPROVED** [review]
LGTM 👍

**Pull request merged**